package.json

```diff
@@ -58,13 +68,15 @@
     "browserstack": "grunt build && sh build/run-browserstack.sh",
     "build": "grunt build",
     "test": "grunt",
-    "coverage": "grunt coverage"
+    "coverage": "grunt coverage",
+    "test:cli": "grunt build && npm link && qunit test/cli/*.js"
```

Comment:
Yeah, by calling `npm link` we can call `qunit` as actual users will after installing it.